server/code_generator.py

```diff
@@ -0,0 +1,70 @@
+import random
+import string
+
+bad_words_file = open('server/bad_words.txt', 'r')
+BAD_WORDS_LIST = bad_words_file.read().split('\n')
+bad_words_file.close()
+
+def generate_code(num_of_chars):
+    """ 
```

Comment:
Personally not a fan of big block comments like this. Your code is very basic and easy to understand, and you have good tests, so feel free to leave them out! Although, if you like them, ignore me and carry on.